.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+# Describe what this PR does #
+
+Provide some context for the reviewer
+
+## Is there anything that requires special attention ##
+
+Do you have any questions? Did you do something clever?
+
+## Related issues ##
+
+Mention any github issues relevant to this PR
```

Comment:
Without making it any longer :) would it make sense to add "Future concerns" section to this template? This is to list items that are not part of the PR and do not impact it's functionality, but are work items that can be taken up in the subsequently.

Reply:
Added